packages/fxa-auth-server/fxa-oauth-server/lib/assertion.js

```diff
@@ -66,6 +66,10 @@ const CLAIMS_SCHEMA = Joi.object({
     .integer()
     .min(0)
     .optional(),
+  'fxa-keysChangedAt': Joi.number()
```

Comment:
More of a documentation nit (or add as part of https://github.com/mozilla/fxa/issues/2508), but I'm not sure we have a location to document our claims schema. Closest I found was https://github.com/mozilla/fxa/blob/eddea169bb45547776ab81779541edc1365686a9/packages/fxa-auth-server/docs/api.md#L3263. Most of the values are self explanatory but some are not.

Reply:
Thanks, I added some docs for other fields (like `fxa-tokenVerifier`) while I was in here.